project-base/storefront/components/Pages/Order/ContactInformation/ContactInformationDeliveryAddress.tsx

```diff
@@ -84,7 +84,7 @@ export const ContactInformationDeliveryAddress: FC = () => {
                     setValue(formMeta.fields.deliveryPostcode.name, deliveryAddress.postcode, { shouldValidate: true });
                 }
             }
-        } else if (deliveryAddressUuid === '') {
+        } else {
```

Comment:
I am not sure if this is good improvement. Just for better type check on line 67 I would leave it. But also since this logic is pretty complex I don't think that connecting those two values here is good idea. I would rather keep them both in the check.